bon-macros/src/builder/builder_gen/mod.rs

```diff
@@ -715,12 +717,59 @@ impl BuilderGenCtx {
             }
         };
 
-        let setter_methods = setters
+        let setget_methods = setgets
             .into_iter()
-            .map(|(setter_methods, _)| setter_methods)
+            .map(|(setter_method, _, maybe_getter_method)| {
+                [setter_method, maybe_getter_method.unwrap_or_default()].concat()
+            })
```

Comment:
Nested concat can be avoided like this:

```suggestion
            .flat_map(|(setter_method, _, maybe_getter_method)| {
                [setter_method].into_iter().chain(maybe_getter_method)
            })
```
This is thanks to the fact that `Option` implements `IntoIterator`